[2025-03-16T03:37:35.863+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:37:35.877+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:37:27.564271+00:00 [queued]>
[2025-03-16T03:37:35.885+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:37:27.564271+00:00 [queued]>
[2025-03-16T03:37:35.886+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2025-03-16T03:37:35.897+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.ingest_raw_data> on 2025-03-16 03:37:27.564271+00:00
[2025-03-16T03:37:35.904+0000] {standard_task_runner.py:72} INFO - Started process 1892 to run task
[2025-03-16T03:37:35.910+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.ingest_raw_data', 'manual__2025-03-16T03:37:27.564271+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmphd2twvf7']
[2025-03-16T03:37:35.911+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask bronze_layer_group.ingest_raw_data
[2025-03-16T03:37:35.961+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:37:27.564271+00:00 [running]> on host 3174348cc1e9
[2025-03-16T03:37:36.042+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.ingest_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T03:37:27.564271+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T03:37:27.564271+00:00'
[2025-03-16T03:37:36.043+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:37:36.045+0000] {ingest_raw_data.py:136} INFO - Scanning for files in topics/tracking.user_behavior.validated/year=2025/month=01...
[2025-03-16T03:37:36.046+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:37:36.047+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:37:36.048+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:37:36.048+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:37:36.520+0000] {ingest_raw_data.py:147} INFO - Loading checkpoint data...
[2025-03-16T03:37:36.612+0000] {ingest_raw_data.py:151} INFO - No new files to process
[2025-03-16T03:37:36.614+0000] {python.py:240} INFO - Done. Returned value was: {'data': [], 'skipped_files': 2}
[2025-03-16T03:37:36.643+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:37:36.643+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.ingest_raw_data, run_id=manual__2025-03-16T03:37:27.564271+00:00, execution_date=20250316T033727, start_date=20250316T033735, end_date=20250316T033736
[2025-03-16T03:37:36.695+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:37:36.718+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T03:37:36.720+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
